openam-server pom.xml - timeout parameter is wrong in cargo step

Hi, I am trying to build OpenAM in an underpowered virtual machine and I am experiencing timeout problems building the openam-server package:

```
[INFO] --- cargo-maven2-plugin:1.7.7:uberwar (default-uberwar) @ openam-server ---
[INFO] Building war: /home/luca/OpenAM/openam-server/target/OpenAM-14.4.3-SNAPSHOT.war
[INFO] 
[INFO] --- cargo-maven2-plugin:1.7.7:start (start) @ openam-server ---
[INFO] [2.ContainerStartMojo] Resolved container artifact org.codehaus.cargo:cargo-core-container-tomcat:jar:1.7.7 for container tomcat8x
[INFO] You did not specify a container home nor any installer. CARGO will automatically download your container's binaries from [http://repo.maven.apache.org/maven2/org/apache/tomcat/tomcat/8.5.43/tomcat-8.5.43.zip].
[INFO] [talledLocalContainer] Tomcat 8.x starting...
...
[INFO] [talledLocalContainer] INFO: Deploying web application archive [/home/luca/OpenAM/openam-server/target/cargo/configurations/tomcat8x/webapps/openam.war]
[INFO] [yer.DeployerWatchdog] Deployable [http://localhost:8080/cargocpc/index.html] failed to finish deploying within the timeout period [120000]. The Deployable state is thus unknown.
[WARNING] [talledLocalContainer] org.codehaus.cargo.container.ContainerException: Deployable [http://localhost:8080/cargocpc/index.html] failed to finish deploying within the timeout period [120000]. The Deployable state is thus unknown.
[ERROR] Starting container [org.codehaus.cargo.container.tomcat.Tomcat8xInstalledLocalContainer@26844abb] failed, now stopping container
```

120 seconds is the default cargo timeout value (https://codehaus-cargo.github.io/cargo/Container+Timeout.html), however, in the `openam-server/pom.xml` file you will find the following lines that suggest that the timeout should be set to 60 seconds:

https://github.com/OpenIdentityPlatform/OpenAM/blob/32b502326a5fd55d476aa8227e300ba3aad206b9/openam-server/pom.xml#L48-L50

These lines however are ignored by maven because the correct syntax needs an extra `<container>` element around the `<timeout>` element:

```
<configuration>
  <container>
    <timeout>60000</timeout>
  </container>
</configuration>
```

The same error is present a few lines later for the container shutdown timeout.

In my case, I had to correct the tag and increase the timeout values to 180000 and 60000 respectively for successfully starting and stopping the container.

--- a/openam-server/pom.xml
+++ b/openam-server/pom.xml
@@ -38,28 +38,32 @@
                 <groupId>org.codehaus.cargo</groupId>
                 <artifactId>cargo-maven2-plugin</artifactId>
                 <extensions>true</extensions>
-				  <executions>
-				    <execution>
-				      <id>start</id>
-				      <phase>pre-integration-test</phase>
-				      <goals>
-				        <goal>start</goal>
-				      </goals>
-				      <configuration>
-				        <timeout>60000</timeout>
-				      </configuration>
-				    </execution>
-				    <execution>
-				      <id>stop</id>
-				      <phase>post-integration-test</phase>
-				      <goals>
-				        <goal>stop</goal>
-				      </goals>
-				      <configuration>
-				        <timeout>20000</timeout>
-				      </configuration>
-				    </execution>
-				</executions>
+                <executions>
+                    <execution>
+                        <id>start</id>
+                        <phase>pre-integration-test</phase>
+                        <goals>
+                            <goal>start</goal>
+                        </goals>
+                        <configuration>
+                            <container>
+                                <timeout>180000</timeout>
+                            </container>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>stop</id>
+                        <phase>post-integration-test</phase>
+                        <goals>
+                            <goal>stop</goal>
+                        </goals>
+                        <configuration>
+                            <container>
+                                <timeout>60000</timeout>
+                            </container>
+                        </configuration>
+                    </execution>
+                </executions>
                 <configuration>
                     <descriptor>src/assemble/merge.xml</descriptor>
                     <archive>

